src/libraries/System.Reflection/tests/CoreCLR/System.Reflection.CoreCLR.Tests.csproj

```diff
@@ -14,5 +14,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\TestExe\System.Reflection.TestExe.csproj" />
+
+    <TrimmerRootDescriptor Include="$(MSBuildThisFileDirectory)..\ILLink.Descriptors.CoreCLR.xml" />
```

Comment:
Since this xml file is only used by this test, can it be moved into this directory?